packages/kbn-monaco/src/register_globals.ts

```diff
@@ -47,17 +37,6 @@ registerLanguage(ESQLLang);
 registerLanguage(ConsoleLang);
 registerLanguage(ConsoleOutputLang);
 
-/**
- * Register custom themes
- */
-registerTheme(ESQL_LIGHT_THEME_ID, buildESQLTheme({ darkMode: false }));
-registerTheme(ESQL_DARK_THEME_ID, buildESQLTheme({ darkMode: true }));
-registerTheme(CONSOLE_THEME_ID, buildConsoleTheme());
-registerTheme(CODE_EDITOR_LIGHT_THEME_ID, buildLightTheme());
-registerTheme(CODE_EDITOR_DARK_THEME_ID, buildDarkTheme());
-registerTheme(CODE_EDITOR_LIGHT_THEME_TRANSPARENT_ID, buildLightTransparentTheme());
-registerTheme(CODE_EDITOR_DARK_THEME_TRANSPARENT_ID, buildDarkTransparentTheme());
-
```

Comment:
All of this is now handled in the actual component implementation of the code editor, see https://github.com/elastic/kibana/pull/203337/files#diff-21b735fba0b5df628e2664a3605f8401d082a4eb32ab0b1ede5fae2db5251030R177-R193